OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoBriefingLogic.cs

```diff
@@ -29,7 +29,8 @@ public GameInfoBriefingLogic(Widget widget, ModData modData, World world)
 			var missionData = world.Map.Rules.Actors[SystemActors.World].TraitInfoOrDefault<MissionDataInfo>();
 			if (missionData != null)
 			{
-				var text = WidgetUtils.WrapText(missionData.Briefing?.Replace("\\n", "\n"), mapDescription.Bounds.Width, mapFont);
+				var briefing = missionData.Briefing != null ? previewWidget.Preview().GetLocalisedString(missionData.Briefing) : "";
```

Comment:
This is removing the `.Replace("\\n", "\n")`, is it no longer needed? Tested on Windows and Linux?

Reply:
https://github.com/OpenRA/OpenRA/pull/21526#discussion_r1759783093

Reply:
Seems like we still use it in [MissionBrowserLogic.cs](https://github.com/OpenRA/OpenRA/pull/21526/files#diff-37e7f2fc75993f2e180e35dfb2ef177b9df17409d7593dba87d0760eef35569e)

Reply:
Do you want me to remove it there as well?